mlrun/db/httpdb.py

```diff
@@ -2673,10 +2638,53 @@ def get_model_endpoint(
                 "metric": metrics or [],
                 "feature_analysis": feature_analysis,
             },
-            headers={"X-V3io-Access-Key": access_key},
         )
         return schemas.ModelEndpoint(**response.json())
 
+    def patch_model_endpoint(
+        self,
+        project: str,
+        endpoint_id: str,
+        attributes: dict,
+    ):
+        """
+        Updates model endpoint with provided attributes.
+
+        :param project: The name of the project.
+        :param endpoint_id: The id of the endpoint.
+        :param attributes: Dictionary of attributes that will be used for update the model endpoint. The keys
+                           of this dictionary should exist in the target table. Note that the values should be
+                           from type string or from a valid numerical type such as int or float.
+
+                           example::
+
+                           # Generate current stats for two features
+                           current_stats = {'tvd_sum': 2.2,
+                                            'tvd_mean': 0.5,
+                                            'hellinger_sum': 3.6,
+                                            'hellinger_mean': 0.9,
+                                            'kld_sum': 24.2,
+                                             kld_mean': 6.0,
+                                            'f1': {'tvd': 0.5, 'hellinger': 1.0, 'kld': 6.4},
+                                            'f2': {'tvd': 0.5, 'hellinger': 1.0, 'kld': 6.5}}
+
+                           # Create attributes dictionary according to the required format
+                           attributes = {
+                                "current_stats": json.dumps(current_stats),
+                                "drift_status": "DRIFT_DETECTED",
+                           }
```

Comment:
I see that you added doc explaining it in here, I would add it also to the actual endpoint.
Also where I can find all possible attributes that can be patched?

Reply:
All the possible attributes can be found under the `ModelEndpoint` object in `api/schemas/model_ednpoints.py`. e.g. `ModelEndpointStatus`:
```
class ModelEndpointStatus(ObjectStatus):
    feature_stats: Optional[dict]
    current_stats: Optional[dict]
    first_request: Optional[str]
    last_request: Optional[str]
    accuracy: Optional[float]
    error_count: Optional[int]
    drift_status: Optional[str]
    drift_measures: Optional[dict]
    metrics: Optional[Dict[str, Metric]]
    features: Optional[List[Features]]
    children: Optional[List[str]]
    children_uids: Optional[List[str]]
    endpoint_type: Optional[EndpointType]
    monitoring_feature_set_uri: Optional[str]
```